scripts/DSRA_outputs2postgres_lfs.py

```diff
@@ -146,11 +146,16 @@ def GetDataframeForScenario(url, repo_list, retrofitPrefix, eqscenario, columnCo
     #dfLosses['sl_BldgT'] = dfLosses['sl_Str'] + dfLosses['sl_NStr'] + dfLosses['sl_Cont']  
     dfLosses = dfLosses.add_suffix("_{}".format(retrofitPrefix))
     dfLosses.rename(columns={"AssetID_{}".format(retrofitPrefix):"AssetID"}, inplace=True)
+
+    eq_url = f'https://raw.githubusercontent.com/OpenDRR/earthquake-scenarios/master/initializations/s_Hazard_{eqscenario}.ini'
```

Comment:
@anthonyfok if we add the ini files to the release assets then we won't have to go through the raw URL. What do you think?